apps/tup-cms/src/taccsite_cms/templates/snippets/js-new-tab-new-window-accessibility.html

```diff
@@ -0,0 +1,23 @@
+
+<script id="new-tab-new-window-accessibility" type="text/javascript">
+
+    // NOTE: Some links have `target="_blank"` even though they should not
+    const pathsOfManualNewTabLinks = document.querySelectorAll(
+        '[target="_blank"]:not([aria-describedby])'
+    );
+
+    // Adds accessibility attribute to links that open in new tab
+    function supportA11y( link ) {
+        const hasA11yNote = link.querySelector('span.sr-only');
+        const hasA11yDesc = link.hasAttribute('aria-describedby');
+        if ( ! hasA11yNote && ! hasA11yDesc ) {
+            link.setAttribute('aria-describedby', 'msg-open-new-window');
```

Comment:
**Ineffectual.** [The `aria-describedby` attribute must refer to an ID of an element with text to describe the link.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)

<details><summary>Screenshot</summary>

| ineffectual during local test | working example on DesignSafe - Accessibility | working example on DesignSafe - HTML |
| - | - | - |
| ![attribute refers to missing element](https://github.com/TACC/tup-ui/assets/62723358/6e8f552d-21dc-480e-a299-48e50740acb8) | ![working example - shows  Description](https://github.com/TACC/tup-ui/assets/62723358/50cb556c-2f61-44ab-88f0-11994572f9b6) |  ![working example - has spans with text](https://github.com/TACC/tup-ui/assets/62723358/dbfc3bb4-3ec6-47d2-ae67-10829088c872) |

</details>
<details><summary>Direction</summary>

The attachment `TUP-575.html` in [TUP-575](https://tacc-main.atlassian.net/browse/TUP-575) has the markup.

</details>

Reply:
[072a4e7](https://github.com/TACC/tup-ui/pull/388/commits/072a4e726c3de6f40999b22620ffb00bc77f47ef)
https://github.com/TACC/tup-ui/pull/388#issuecomment-1847547885